v1/endpoints/enriched_metadata/programs.yml

```diff
@@ -4,7 +4,7 @@ info:
     This endpoint provides metadata about Programs.
 
     A program is something that can be shown.
-    It could be a television show (like "Friends") or a movie (like "Die Hard") that appears on a Channel at a given time in a BroadcastSlot.
+    It could be an episode of a television show (like "Friends") or a movie (like "Die Hard") that appears on a Channel at a given time in a BroadcastSlot.
```

Comment:
Good spot - I like the idea.  Episode is a named entity in our UBL so please could you capitalize it and I'll merge the request.  I just try to capitalize named entities to highlight them (even if it is gramatically incorrect)

Reply:
I don't think that's necessary as it's not specifically referring to that entity. This was just to clarify that a Program is different from what we refer to as "Series", but I will capitalize it if you insist.

Reply:
Please do.  If it is an episode of a series then it is an instance of an Episode entity.